packages/widgets/src/calendar/calender-day.tsx

```diff
@@ -59,20 +64,19 @@ export const CalendarDay = ({ date, events, disabled }: CalendarDayProps) => {
   );
 };
 
-interface DayIndicatorProps {
-  size: number;
-  color: string;
+interface ArrIndicatorProps {
   events: CalendarEvent[];
-  children: ReactNode;
-  position: IndicatorProps["position"];
 }
 
-const DayIndicator = ({ size, color, events, children, position }: DayIndicatorProps) => {
-  if (events.length === 0) return children;
-
+const NotificationIndicator = ({ events }: ArrIndicatorProps) => {
```

Comment:
Interface suggests that this component only applies for Arr apps. But this is not the case. Rename ``NotificationIndicator`` or ``ArrIndicatorProps``